devolutions-host/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "devolutions-host"
+version = "2024.3.1"
```

Comment:
The version number is currently 2024.3.2.
Also, can you update the PowerShell script found in tools called bump-version.ps1 if the intention is to keep everything in sync?